duckduckgo/iOS
# Navigation structure updates

Reviewer: Caine
Asana: https://app.asana.com/0/276208855376242/290598814185289

**Description**:
Updates app to use new navigational structure:
- simplifies home screen and adds animation
- consolidates separate omnibar designs into one
- adds browsing menu to omni bar & moves settings into menu
- updates tab switcher to new design
- updates app structure so that the "home" screen is no longer a tab

**Steps to test this PR**:
- Run the app
- Note thae home screen UI changes. In particular test activating and dismissing the search bar
- Check that a new menu appears in the browser view and that the refresh, copy, bookmark and settings options all work as expected
- Note the tab switcher UI changes. Check that tabs can be added, and individual tab can be closed and that the fire button now clears everything but remains on the switcher

###### Reviewer Checklist:

###### PR DRI Checklist:
